ams/server/src/main/java/com/netease/arctic/server/dashboard/PaimonTableDescriptor.java

```diff
@@ -154,12 +162,58 @@ public List<DDLInfo> getTableOperations(AmoroTable<?> amoroTable) {
 
   @Override
   public List<PartitionBaseInfo> getTablePartition(AmoroTable<?> amoroTable) {
```

Comment:
`getTablePartitions` may be a better method name.

Reply:
yeah